luigi/execution_summary.py

```diff
@@ -377,33 +437,40 @@ def _summary_format(set_tasks, worker):
         if len(ext_workers) == 0:
             str_output += '\n'
         str_output += 'Did not run any tasks'
-    smiley = ""
-    reason = ""
+    summary_text_one_line = _progress_summary(_tasks_status(set_tasks))
+    str_output += "\n{0}".format(summary_text_one_line)
+    if num_all_tasks == 0:
+        str_output = 'Did not schedule any tasks'
+    return str_output
+
+
+def _progress_summary(status_code):
+    """
+    Given a status_code of type LuigiStatusCode which has a tuple value, returns a one line summary
+    """
+    summary_text_one_line = "This progress looks {0} because {1}".format(*status_code.value)
+    return summary_text_one_line
```

Comment:
Why not just directly?
 
    return  "This progress looks {0} because {1}".format(*status_code.value)